packages/web-components/src/components/ic-accordion/ic-accordion.stories.mdx

```diff
@@ -0,0 +1,374 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs";
+import { html } from "lit-html";
+import readme from "./readme.md";
+
+<Meta title="Web Components/Accordion" component="ic-accordion" />
+
+<Description markdown={readme} />{" "}
+
+### Default
+
+<Canvas>
+  <Story name="Default">
+    {html` <ic-accordion heading="Accordion 1">
+        <ic-typography variant="body">
+          This is an example of the main body text.
+        </ic-typography>
+      </ic-accordion>
+      <ic-accordion heading="Accordion 2">
+        <ic-typography variant="body">
+          This is an example of the main body text.
+        </ic-typography>
+      </ic-accordion>`}
+  </Story>
+</Canvas>
+
+### Icon
+
+<Canvas>
+  <Story name="Icon">
+    {html` <ic-accordion heading="Accordion 1">
+        <svg
```

Comment:
The icon isn't quite aligned to the centre of the text vertically:

<img src="https://github.com/mi6/ic-ui-kit/assets/111882109/16968d98-166a-4144-b471-1d1343c54825" width="200">

Also, just thinking if someone tries to pass in an icon which has a random height and width - should there be a set height and width applied to the icon so whatever size icon they pass in, it always appears at the correct size?

Thinking maybe an 18px height + width on the SVG and 24px on the `icon-container` div to match the designs? :)